app/javascript/components/retirement-form/index.jsx

```diff
@@ -0,0 +1,88 @@
+import React, { useEffect, useState } from 'react';
+
+import MiqFormRenderer from '@@ddf';
+import createSchema from './retirement-form.schema';
+import { API, http } from '../../http_api';
+import handleFailure from '../../helpers/handle-failure';
+
+const RetirementForm = ({ retirementID }) => {
+  console.log(retirementID)
+  const id = retirementID.split('"').filter(Number);
```

Comment:
I don't really understand this :confused:

Reply:
retirementID is a string that looks like this: ["2"]. I was splitting the string at " to turn it into an array to then filter by numbers within it. (resulting in an array of id numbers). Since it should only receive one id, I changed it to use regex to filter the id.